crates/js-sys/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use std::mem;
 use std::str;
 use std::str::FromStr;
 
+pub use wasm_bindgen;
```

Comment:
```suggestion
pub extern crate wasm_bindgen;
```
This is how reexports are usually done.

Reply:
The only re-export I have worked was in [`wai-bindgen-wasmtime`](https://github.com/wasmerio/wai/blob/main/crates/wasmtime/src/lib.rs#L8) and [`wai-bindgen-wasmer`](https://github.com/wasmerio/wai/blob/main/crates/wasmer/src/lib.rs#L8). Also [this link](https://www.lurklurk.org/effective-rust/re-export.html) suggested using the `pub use dependency_crate` syntax.

But if you want `pub extern crate wasm_bindgen;`, go for it, it's your repo. I checked the "allow maintainers to make changes", so that means you can put the change into my fork of the repo? I'm not super familiar with this on GitHub.

Reply:
I recently found out that `pub extern` and `pub use` basically emit the same things on docs.rs.